meetings/2024-12-10-taskforce-meeting.md

```diff
@@ -0,0 +1,70 @@
+# 2024-12-10: ActivityPub Trust and Safety Task Force Meeting
+
+### Participating
+
+- Lisa Dusseault (@lisarue@mastodon.geekery.org)
+- Darius Kazemi (@darius@friend.camp)
+- Evan Prodromou <acct:evan@cosocial.ca>
+- Jaz-Michael King (IFTAS, @jaz@mastodon.iftas.org)
+- Emelia Smith (@thisismissem@hachyderm.io)
+- James Smith (Manyfold)
+- Andreas Savvides (Threads)
+- Mehdi Benadel (@mehdi_benadel@mastodon.balamb.fr)
+- [TallTed](https://github.com/TallTed) // Ted Thibodeau (he/him) (https://mastodon.social/@TallTed)
+
+### Agenda
+
+1. IP Protection Note Reminder
+   * Anyone can participate in these calls. However, all substantive contributors to any Taskforce Work Items must be members of the Social Web CG with full IPR agreements signed. https://www.w3.org/community/socialcg/join
+   * To contribute to Work Items: ensure you have a W3 account: https://www.w3.org/accounts/request, and sign the W3C Community Contributor License Agreement (CLA): https://www.w3.org/community/about/agreements/cla/
+
+2. Reminder of Code of Conduct
+
+3. Introductions, if necessary
+
+4. Review and merge Update README.md to include scope of work #39 if it hasn't already been approved and merged.
+
+5. Meeting schedule over the holiday season: PROPOSAL: skip the scheduled meeting on 24th December, reconvene 7th January.
+
+6. Discussion
+
+
+### Minutes
+
+* discussion of initial report
+    * https://github.com/swicg/activitypub-trust-and-safety/issues/32
+    * Evan volunteers to provide user stories on content warnings, labels, and annotations
+    * Emelia notes that reading of the history behind content warnings is strongly recommended: https://shelraphen.com/on-content-warnings/
+    * "Best practices" section https://github.com/swicg/activitypub-trust-and-safety/issues/32
+        * Emelia: goal is to outline features we think would be the bare minimum things people should do in AP software for T&S
+        * Evan: If we are doing different sections, do you want volunteers to take on different sections of the report?
+        * Emelia: yes
+        * _Addendum: the initial report is intended to have multiple sections and volunteers to collaboratively write those sections, this will become clearer with the initial scaffolding of the initial report_
```

Comment:
```suggestion
        * _Addendum: the initial report is intended to have multiple sections and volunteers to collaboratively write those sections; this will become clearer with the initial scaffolding of the initial report._
```